prac_05/wimbledon.py

```diff
@@ -0,0 +1,45 @@
+"""
+Wimbledon
+Estimate: 90 minutes
+Actual:   105 minutes
+"""
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    records = get_records(FILENAME)
+    champion_to_count = count_the_champions(records)
+    countries = get_the_country_of_champions(records)
+    for champion, count in champion_to_count.items():
+        print(f"{champion} {count}")
+    print(f"\nThese {len(countries)} have won Wimbledon:")
+    for country in countries:
+        print(country, end=", ")
+
+
+def get_the_country_of_champions(records):
+    countries = set()
+    for record in records:
```

Comment:
Maybe put a comment to explain or have a quick definition what this for loop for?